### ISSUE TITLE
after update can't even use the engine

### ISSUE THREAD

**Issue by idg32** [godotengine/godot#57686]
### Godot version

3.4.2

### System information

windows 11

### Issue description

welp, it says some garbage before closing the app before I can read it...

I launch, prints stuff, closes 2 seconds later... no clue what it's saying

### Steps to reproduce

just launch the engine

### Minimal reproduction project

you don't even need a project

**Comment by Calinou** [comment]
> I launch, prints stuff, closes 2 seconds later... no clue what it's saying

Can you record or a video or take a screenshot? Without this information, we can't do anything.

**Comment by idg32** [comment]
https://user-images.githubusercontent.com/95511222/152664172-be84dedb-9328-4d2f-851e-2a1c7861a9c7.mp4

yar :)

**Comment by idg32** [comment]
![image](https://user-images.githubusercontent.com/95511222/152664359-9cd4b41a-9fc5-4c93-a395-ddb4b952f6a5.png)

**Comment by idg32** [comment]
this happens on steam and standalone, haven't tried compiled

**Comment by idg32** [comment]
so if I run in video version gles2 it works. but I had to run as admin to get my gles3 project up and running

**Comment by idg32** [comment]
it produces this without changing the game's settings 
![image](https://user-images.githubusercontent.com/95511222/152664514-ac73031f-53d5-42b8-87b0-1dc4329017ce.png)

**Comment by clayjohn** [comment]
Can you please provide more information that we can use to identify the source of the bug? For example, what hardware are you using (CPU and GPU model)? Were you able to run any previous versions of Godot on this hardware? If so, what versions? Have you tried updating your GPU drivers recently?

**Comment by idg32** [comment]
YES all drivers are up to date. 1030 pheonix gtx asus 2gb. i5 10700 cpu 2.9ghz, adn 12 gb ram. I was perfectly deving on 3.4.1 :)

**Comment by clayjohn** [comment]
Did you recently install anything that would inject itself into the graphics layer? Any Reshade mods or things like that? What about any recent change in settings to the NVidia control panel, is there any override settings that may be affecting things?

I ask because the errors you report above reference functions that are not used at all in Godot. So it looks like something else may be interfering.

**Comment by idg32** [comment]
I have GPU tweak from ASUS, and GE FORCE, aside from C.P.
But I've not installed anything since prior to the update

**Comment by idg32** [comment]
would rtx voice do it?

**Comment by Calinou** [comment]
There's a Reddit post from someone who had the same "ANOMALY: meaningless REX failed" message: https://www.reddit.com/r/godot/comments/iowng0/meaningless_rex_prefix/

**Comment by ulvaske** [comment]
OSD

**Comment by idoadler** [comment]
Same error happens to me if ASUS GPU TweakII runs in the background.

**Comment by WizPip** [comment]
> Same error happens to me if ASUS GPU TweakII runs in the background.

I've been having issues with Windows 11 randomly closing programs and not letting me install / uninstall anything. I've gone all round the houses and was just about to reset Windows and prepared myself for hours of fun reinstalling everything, but you just saved me with this thing - GPU Tweak was causing it!!
